### PR TITLE
[Documentation] Add changes on recognizers-text utterances to the package description

### PR DESCRIPTION
**Pull request by Aliandi**
### Description
This PR adds a temporal description to the `.csproj` of `Microsoft.Bot.Builder` and `Microsoft.Bot.Builder.Dialogs` that includes the changes that come with the Recognizers-Text version upgrade.

### Changes made
- Add description to `Microsoft.Bot.Builder.csproj`
- Add description to `Microsoft.Bot.Builder.Dialogs`

### Result
This is what the preview looks like when the package is generated:

imagen

_Note: This is a temporal solution, in the future the release note should be changed. If the NuGet team adds the option to [add markdown](https://github.com/NuGet/Home/wiki/Packaging-Documentation-within-the-nupkg), we would probably want to link the changes instead of writing them down. However, this will do for an initial version upgrade._

### CODE REVIEW

File: libraries/Microsoft.Bot.Builder.Dialogs/Microsoft.Bot.Builder.Dialogs.csproj [hunk 1]
```diff
@@ -18,7 +18,40 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <Description>This library implements .NET Simple Dialog classes </Description>
+    <Description>
+      This library implements .NET Simple Dialog classes
```
Comment (GasparAcevedoZainSouthworks) [inline libraries/Microsoft.Bot.Builder.Dialogs/Microsoft.Bot.Builder.Dialogs.csproj]: Is this line necessary? As this is the same line from the _Summary_

File: libraries/Microsoft.Bot.Builder/Microsoft.Bot.Builder.csproj [hunk 2]
```diff
@@ -19,7 +19,39 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <PackageId>Microsoft.Bot.Builder</PackageId>
-    <Description>Library for building bots using Microsoft Bot Framework Connector</Description>
+    <Description>
+	Library for building bots using Microsoft Bot Framework Connector
```
Comment (GasparAcevedoZainSouthworks) [inline libraries/Microsoft.Bot.Builder/Microsoft.Bot.Builder.csproj]: Same here about the comment and the `Summary`

File: libraries/Microsoft.Bot.Builder.Dialogs/Microsoft.Bot.Builder.Dialogs.csproj (hunk 1 above)
Comment (Aliandi) [reply libraries/Microsoft.Bot.Builder.Dialogs/Microsoft.Bot.Builder.Dialogs.csproj]: I am not sure it is, but it was like that before so I don't know if we should change it. What do you think?

Comment (GasparAcevedoZainSouthworks) [reply libraries/Microsoft.Bot.Builder.Dialogs/Microsoft.Bot.Builder.Dialogs.csproj]: OK, just leave it as it is.

**Pull request merged**